drip/templates/admin/drip/change_form.html

```diff
@@ -1,63 +1,80 @@
-{% extends "admin/change_form.html" %}
-{% load i18n static admin_list %}
-{% load admin_urls %}
-
-{% block object-tools-items %}
-  <li><a href="{% url 'admin:drip_timeline' original.id 4 7 %}" class="">View Timeline</a></li>
-  <li><a href="history/" class="historylink">{% trans "History" %}</a></li>
-  {% if has_absolute_url %}<li><a href="../../../r/{{ content_type_id }}/{{ object_id }}/" class="viewsitelink">{% trans "View on site" %}</a></li>{% endif%}
-{% endblock %}
-
-{% block after_related_objects %}
+{% extends "admin/change_form.html" %} {% load i18n static admin_list %} {% load
```

Comment:
What about this one-line tags?

Reply:
have no idea. This is from the previous maintainers